packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,224 @@
+| [Identity Proofing](https://www.okta.com/integrations/?category=identity-proofing) | Enables user self-verification to improve identity assurance and approve access for authorized individuals using document-based and/or knowledge-based proofs |
+| [Identity Governance and Administration (IGA)](https://www.okta.com/integrations/?category=identity-governance-and-administration-iga) | Provides the right people access to the right tools at the right time |
+| [Zero Trust](https://www.okta.com/integrations/?category=zero-trust) | Enables secure access for users regardless of their location, device, or network |
+| [Bot or Fraud Detection](https://www.okta.com/integrations/?category=bot-or-fraud-detection) | Provides protection from inauthentic, forged, or otherwise fradulent attempts to register, sign-in, recover, or perform identity-related transactions. Most integrations in this use case are API service integrations that send risk signals to Okta using OAuth 2.0. |
+| [Multi-factor Authentication (MFA)](https://www.okta.com/integrations/?category=multi-factor-authentication-mfa) | Provides an additional layer of security with multifactor authentication for an organization’s cloud, mobile, and web apps |
+| [Risk Signal Sharing](https://www.okta.com/integrations/?category=risk-signal-sharing)| Provides enriched context on clients, apps, users, and other first-party subjects to augment and inform Okta's adaptive authentication and authorization decisions |
+| [Apps for Good](https://www.okta.com/integrations/?category=apps-for-good) | Pre-built integrations with leading apps that enable your workforce to give back |
+
+## Test account guidelines
+
+You need to create a test account on your app so that the OIN team can use it to test and verify your integration. The test account details and credentials are specified  in the **General Settings** > **Test Account** fields from the OIN Manager.
+
+??? Work on:
+
+We need a test user with admin access who has the rights to configure SSO for SAML/SCIM. Even if your support team has configured SSO or SCIM, we need access to verify if users and/or groups on the application end are created by SCIM Provisioning or SAML/OIDC (JiT).
+
+In general, the test account allows the OIN team to verify that your integration flow works as expected for your use case.
+
+We prefer isvtest@okta.com to be a test user. If you can't use this email to create a test user, you can provide a specific account in a different domain.
+
+??? Perhaps remove/merge with the above ???
+
+Depending on the use case of our app, the permissions and roles for your test account are varied. For example:
+
+* For an SSO integration, you can provide a regular, non-admin user test account for the OIN team to test Single Sign-On with your app.
```

Comment:
Question for Abdullah, but can we just ask for an admin user since you can also test SSO for an admin user?

If we want to remove the nuance for each type can we say that the user should have permission to configure the integration?

Reply:
I've updated this section to incorporate @mohammadabdullah-okta's feedback on user permissions.